Generator/Templates/GObject/class.sbntxt

```diff
@@ -38,65 +38,58 @@ namespace {{ namespace }}
     
     public partial class {{ name }} {{ write_inheritance }}
     {
-{{if !fundamental}}
+{{~ if !fundamental ~}}
         private static readonly TypeDescriptor GTypeDescriptor = TypeDescriptor.For("{{ type_name }}", {{ name }}.Native.get_type);
 
         #region Signals
         
-        {{- if signals_have_event_args }}
-        
+        {{~ if signals_have_event_args ~}}
         ///<summary>
         /// Indexer to connect with an SignalHandler<{{name}}>
         ///</summary>
         public SignalHandler<{{ name }}> this[Signal<{{ name }}> signal]
         {
             set => signal.Connect(this, value, true);
         }
-        {{ end }}
-
-        {{- for $signal in signals
-
-include 'signal.sbntxt' $signal '        ' name
-
-        end ~}}
+        {{~ end ~}}
+        {{~ for $signal in signals }}
+        {{ include 'signal.sbntxt' $signal name }}
+        {{ end ~}}
         
         #endregion
         
-        {{- if has_parent}} 
-        
+        {{~ if has_parent ~}} 
         #region Constructors
         
         protected internal {{ name }}(IntPtr ptr) : base(ptr) {}
         protected internal {{ name }}(params ConstructParameter[] properties) : base(properties) {}
         
         #endregion
         
-        {{- end }}
-{{- end }}
-
+        {{~ end ~}}
+{{~ end ~}}
         #region Methods
```

Comment:
I think it's better to rename this region to `Native`

Reply:
Done